arbnode/dataposter/externalsignertest/externalsignertest.go

```diff
@@ -114,7 +124,23 @@ func NewServer(t *testing.T) (*http.Server, *SignerAPI) {
 		}
 	})
 
-	return httpServer, s
+	return &SignerServer{httpServer, s}
+}
+
+func (s *SignerServer) URL() string {
+	port := strings.Split(s.Addr, ":")[1]
+	return fmt.Sprintf("https://localhost:%v", port)
```

Comment:
```suggestion
	return fmt.Sprintf("https://%v", s.Addr)
```
Here as well.

Reply:
Actually, this one needs to use "localhost" to match the TLS certificate. If I replace it with s.Addr, (which contains "127.0.0.1" it refuses all of the connections from clients.